src/synced_services/toggl/toggl_synced_service.ts

```diff
@@ -156,29 +156,37 @@ export class TogglTrackSyncedService implements SyncedService {
   // ***********************************************************
 
   private async _getAllProjects(): Promise<ServiceObject[] | boolean> {
+    const projects: ServiceObject[] = [];
     let response;
-    try {
-      response = await this._retryAndWaitInCaseOfTooManyRequests(
-          superagent
-              .get(this._projectsUri)
-              .auth(this._serviceDefinition.apiKey, 'api_token')
-      );
-    } catch (ex: any) {
-      this.handleResponseException(ex, 'getAllProjects');
-      return false;
-    }
 
+    const queryParams = {
+      page: 1,
+      per_page: 200,
+    };
 
-    const projects: ServiceObject[] = [];
+    do {
+      try {
+        response = await this._retryAndWaitInCaseOfTooManyRequests(
+            superagent
+                .get(this._projectsUri)
+                .auth(this._serviceDefinition.apiKey, 'api_token')
+                .query(queryParams)
+        );
+      } catch (ex: any) {
+        this.handleResponseException(ex, 'getAllProjects');
+        return false;
+      }
 
-    response.body?.forEach((project: never) => {
-      projects.push(
-        new ServiceObject(
-          project['id'],
-          project['name'],
-          this._projectsType,
-        ));
-    });
+      response.body?.forEach((project: never) => {
+        projects.push(
+            new ServiceObject(
+                project['id'],
+                project['name'],
+                this._projectsType,
+            ));
+      });
+      queryParams.page += 1;
+    } while (response.body?.length > 0)
```

Comment:
Lepší by bylo využít nějaký atribut total_pages, pokud se takový vrací, takto je to jeden request navíc, ale pokud nic takového Toggl neposílá, tak OK